### ISSUE TITLE
Warning: Source file does not specify required compiler version!

### ISSUE THREAD

**Issue by ethernomad** [nexusdev/dapple-test#1]
```
Testing...
<stdin>:2:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".

^
```

I don't know why this warning appears, because all the contracts and test.sol declare the required compiler version.

**Comment by apmilen** [comment]
does the compiler say which file is causing this warning?

**Comment by mhhf** [comment]
This was caused by the way we test wether the native solc compiler is available. Is solved in then next version and will be included in dapple 0.8.4 today or tomorrow.

**Comment by mhhf** [comment]
Closing as this is resolved.
